Sources/ActionSheet/MediaMoreOptionsActionSheet.swift

```diff
@@ -84,6 +84,13 @@ extension MediaMoreOptionsActionSheet: MediaPlayerActionSheetDataSource {
             if $0 == .interfaceLock {
                 cellModel.accessoryType = .toggleSwitch
                 cellModel.viewToPresent = nil
+            } else if $0 == .playback {
+                if let playbackView = Bundle.main.loadNibNamed("PlaybackSpeedView", owner: self, options: nil)?.first
```

Comment:
trying to instantiate the nib with `nib.instantiate(name:owner:options)` also fails. I think this crash in general might be due to the fact that in the `PlaybackSpeedView`'s `awakeFromNib` method, a `loadNib` function is called that itself calls the `nib.instantiate(name:owner:options)` method. This can be found at line `56` in the `PlaybackSpeedView.swift` file.